projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -451,11 +451,19 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 		update_user_option( get_current_user_id(), $option_name, $variation, true );
 
 		$is_enabled = 'treatment' === $variation;
-		return $is_enabled;
+	} elseif ( isset( $data['variations'] ) && ! isset( $data['variations'][ $experiment_name ] ) ) {
```

Comment:
We can improve/simplify the checks by:
````
if ( isset( $data['variations'][ $experiment_name ] ) ) {
    $variation = $data['variations'][ $experiment_name ];
    update_user_option( get_current_user_id(), $option_name, $variation, true );
    $is_enabled = 'treatment' === $variation;
} elseif ( isset( $data['variations'] ) ) {
  /**
   * If the variations array is set but the variation value is null chances are this is an a11n (since ExPlat returns null for a12s).
   * 
   * We want to set the treatment for all a12s.
   */
    update_user_option( get_current_user_id(), $option_name, 'treatment', true );
    $is_enabled = true;
} else {
    $is_enabled = false;
}
````

Reply:
Yeah, that could work too.